ADD_SAMPLE_INTO_DEV_KIT.md

```diff
@@ -12,7 +12,10 @@ For new samples, use `dev` as the target branch for pull requests. The `dev` bra
 
 ## Testing
 
-TBD
```

Comment:
Larger design question:

Why do we have two extensions instead of an option in the Dev Kit to use a different repo branch?

Reply:
This is a temporary solution. After the migration is done and GA version of Dev Kit released. Users can use the store version with env variable set to directly test sample gallery. No need to download and install a test package.